std/file.d

```diff
@@ -2318,7 +2319,7 @@ private bool ensureDirExists(in char[] pathname)
  * Throws: $(D FileException) on error.
  */
 
-void mkdirRecurse(in char[] pathname)
+void mkdirRecurse()(in char[] pathname)
```

Comment:
>Of course, even changing a single attribute is an API change

Being pedantic here, but isn't adding something like `nothrow` an ABI and not an API change?